.travis.yml

```diff
@@ -12,6 +12,7 @@ script:
   - go generate -run="genmock" -x github.com/google/trillian/...
   - go build -v ./...
   - go test -v ./...
+  - ./integration/map_integration_test.sh
```

Comment:
It would be good to have a base for log / map Integration tests in Go. Maybe for later.